### DISCUSSION TITLE
How to paste image into markdown

### DISCUSSION THREAD

**Discussion by chenxizhang** [community/community#7962] [Codespaces]
We can do this by the extension "markdown paste" or other similar, but we can't make it in codespace.

**Comment by asciimike** [comment]
Does this fail in both the web editor as well as VSCode Desktop? If it's only the former, is it possible that the browser is blocking paste, for example?

**Reply by chenxizhang** [reply]
It works fine in VSCode desktop.

In the browser, I can copy/paste text, but the extension "markdown paste"(usually use to paste image) never work without any error message.
